tests/sqllogictests/suites/base/11_data_type/11_0006_data_type_decimal

```diff
@@ -137,3 +137,62 @@ query I
 SELECT ANY(CAST(2.34 AS DECIMAL(6, 2)))
 ----
 2.34
+
+statement ok
+drop table if exists t
+
+statement ok
+drop table if exists t1
+
+statement ok
+create table t(c1 decimal(38,2))
+
+statement ok
+insert into t(c1) select CAST(2.34 AS DECIMAL(6, 2))
+
+statement ok
+insert into t(c1) select CAST(-2.34 AS DECIMAL(6, 2))
+
+statement ok
+insert into t(c1) select CAST(0 AS DECIMAL(6, 2))
+
+query I
+select * from t order by c1 desc;
+----
+2.34
+0.00
+-2.34
+
+query I
+select * from t order by c1 asc;
+----
+-2.34
+0.00
+2.34
+
+statement error 1067
```

Comment:
```sql
mysql> create table t(c1 decimal(38,2));
Query OK, 0 rows affected (0.14 sec)

mysql> insert into t(c1) select CAST(0 AS DECIMAL(6, 2));
Query OK, 1 row affected (0.10 sec)

mysql> select * from t limit 1;
+------+
| c1   |
+------+
| 0.00 |
+------+
1 row in set (0.05 sec)
Read 1 rows, 16.00 B in 0.009 sec., 106.72 rows/sec., 1.67 KiB/sec.

mysql> select * from t limit 0;
Empty set (0.04 sec)
Read 0 rows, 0.00 B in 0.009 sec., 0 rows/sec., 0.00 B/sec.

mysql> select * from t order by c1 limit 1;
+------+
| c1   |
+------+
| 0.00 |
+------+
1 row in set (0.04 sec)
Read 1 rows, 16.00 B in 0.010 sec., 103.24 rows/sec., 1.61 KiB/sec.

mysql> select * from t order by c1 limit 0,1;
+------+
| c1   |
+------+
| 0.00 |
+------+
1 row in set (0.04 sec)
Read 1 rows, 16.00 B in 0.010 sec., 99.08 rows/sec., 1.55 KiB/sec.

mysql> insert into t(c1) select CAST(2.3 AS DECIMAL(6, 2));
Query OK, 1 row affected (0.10 sec)

mysql> select * from t order by c1 limit 0,1;
ERROR 1105 (HY000): Code: 1067, displayText = Unable to get the colStats by ColumnId: 0(while in query pipeline init).
mysql> select * from t order by c1 limit 1,1;
+------+
| c1   |
+------+
| 2.29 |
+------+
1 row in set (0.04 sec)
Read 2 rows, 32.00 B in 0.009 sec., 225.66 rows/sec., 3.53 KiB/sec.

mysql> select * from t order by c1 limit 2,1;
Empty set (0.04 sec)
Read 2 rows, 32.00 B in 0.009 sec., 221.59 rows/sec., 3.46 KiB/sec.

```

> select * from t order by c1 limit 0,1;
ERROR 1105 (HY000): Code: 1067, displayText = Unable to get the colStats by ColumnId: 0(while in query pipeline init).

This maybe a bug.